client/context/context.go

```diff
@@ -87,14 +90,16 @@ func createCertifier() tmlite.Certifier {
 	if nodeURI == "" {
 		errMsg.WriteString("node ")
 	}
-	// errMsg is not empty
+
 	if errMsg.Len() != 0 {
-		panic(fmt.Errorf("can't create certifier for distrust mode, empty values from these options: %s", errMsg.String()))
+		panic(fmt.Errorf("failed to create certifier for distrust mode; empty values from these options: %s", errMsg.String()))
```

Comment:
It is not proper to use panic as error handler in cli command. Instead, how about print guide message to help users to correct command line and just end the program without pring call stack?